html/main-page.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Дизайн-студия "Нердс"</title>
+</head>
+<body>
+<header class="main-header">
+<nav>
+<a href="blank.html">
```

Comment:
```suggestion
<a>
```
Это текущая страница, поэтому `href` убираем.